scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,229 @@
+    2. Create a non-admin user with username "b" and give the user "submit
+    question" rights.
+    3. Populate sample lessons for translations. This is achieved by reproducing
+    the "Load dummy new structures data" admin action.
+    4. Add the topics linked to the lessons above to a classroom. This is needed
+    for the topics, if published, to show up in the topic selectors and for the
+    linked opportunities to show up in the "Submit Question" tab.
+"""
+
+from __future__ import annotations
+
+import hashlib
+import json
+import os
+
+from core import feconf
+from core.constants import constants
+
+import firebase_admin
+from firebase_admin import auth as firebase_auth
+
+import requests
+from typing import Any, Dict, List
+
+FIREBASE_AUTH_EMULATOR_HOST = 'localhost:%s' % feconf.FIREBASE_EMULATOR_PORT
+os.environ['FIREBASE_AUTH_EMULATOR_HOST'] = FIREBASE_AUTH_EMULATOR_HOST
+FIREBASE_SIGN_IN_URL = (
+    'http://' + FIREBASE_AUTH_EMULATOR_HOST +
+    '/identitytoolkit.googleapis.com/v1/accounts:signInWithPassword')
+
+SUPER_ADMIN_EMAIL = feconf.ADMIN_EMAIL_ADDRESS
+SUPER_ADMIN_USERNAME = 'a'
+SUPER_ADMIN_ROLES = [feconf.ROLE_ID_CURRICULUM_ADMIN,
+        feconf.ROLE_ID_TRANSLATION_ADMIN, feconf.ROLE_ID_QUESTION_ADMIN]
+
+CONTRIBUTOR_EMAIL = 'contributor@example.com'
+CONTRIBUTOR_USERNAME = 'b'
+
+CLASSROOM_NAME = 'math'
+CLASSROOM_URL_FRAGMENT = 'math'
+
+
+class ContributorDashboardDebugInitializer():
+    """Client-side requests to help develop for the contributor dashboard.
+
+    Attributes:
+        session: object(Session). The requests.Session object to send requests
+            to the Oppia server.
+        base_url: str. The base url of the Oppia server.
+        csrf_token: str. The csrf token of the current session.
+    """
+
+    def __init__(self, base_url: str) -> None:
+        self.session = requests.session()
+        self.base_url = base_url
+        self.csrf_token = ''
+
+    def populate_debug_data(self) -> None:
+        """Populate sample data to help develop for the contributor
+        dashboard.
+        """
+        firebase_admin.initialize_app(
+            options={'projectId': feconf.OPPIA_PROJECT_ID})
+
+        self._sign_up_new_user(SUPER_ADMIN_EMAIL, SUPER_ADMIN_USERNAME)
+        self._sign_up_new_user(CONTRIBUTOR_EMAIL, CONTRIBUTOR_USERNAME)
+
+        self._sign_in(SUPER_ADMIN_EMAIL)
+        self.csrf_token = self._get_csrf_token()
+        self._assign_admin_roles(SUPER_ADMIN_ROLES, SUPER_ADMIN_USERNAME)
+        self._add_submit_question_rights(CONTRIBUTOR_USERNAME)
+        self._generate_sample_new_structures_data()
+        self._add_topics_to_classroom(CLASSROOM_NAME, CLASSROOM_URL_FRAGMENT)
+
+    def _sign_up_new_user(self, email: str, username: str) -> None:
+        """Sign up a new user based on email and username. The password is
+        generated automatically from email.
+        """
+        password = hashlib.md5(email.encode('utf-8')).hexdigest()
```

Comment:
Please leave a comment explaining why generating a password by MD5-hashing an email (which is usually very bad) is okay in this situation

Even better, you could just use the email as the password to avoid giving any impression that the MD5 hash adds security properties